cdc/sink/dmlsink/mq/mq_dml_sink.go

```diff
@@ -196,8 +195,11 @@ func (s *dmlSink) WriteEvents(txns ...*dmlsink.CallbackableEvent[*model.SingleTa
 			// We already limit the memory usage by MemoryQuota at SinkManager level.
 			// So it is safe to send the event to a unbounded channel here.
 			s.alive.worker.msgChan.In() <- mqEvent{
-				key: codec.TopicPartitionKey{
-					Topic: topic, Partition: index, PartitionKey: key,
+				key: model.TopicPartitionKey{
+					Topic:          topic,
+					Partition:      index,
+					PartitionKey:   key,
+					TotalPartition: partitionNum,
```

Comment:
Is it necessary to pass in `TotalPartition`? 

Bootstrap is sent to the first partition.